### ISSUE TITLE
Enhancement: Add more Cursors

### ISSUE THREAD

**Issue by juandavidcruzs** [processing/p5.js#3243]
#### Nature of issue?

- [ ] Found a bug
- [x] Existing feature enhancement
- [ ] New feature request

#### Most appropriate sub-area of p5.js?

- [ ] Color
- [x] Core/Environment/Rendering
- [ ] Data
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Other (specify if possible)

#### Feature enhancement details:

There are a lot of cursors as seen on: [CSS Cursor Property W3](https://www.w3schools.com/cssref/pr_class_cursor.asp)

However I found that P5.js only has (ARROW, CROSS, HAND, MOVE, TEXT, or WAIT, or path for image)

In `constants.js` we could add new constants for new cursors such as all the resize cursors:

```javascript
/**
 * @property {String} N_RESIZE
 * @final
*/
N_RESIZE: 'n-resize',
```

```javascript
/**
 * @property {String} S_RESIZE
 * @final
*/
S_RESIZE: 's-resize',
```

etc.

**Comment by SphaeriCactus** [comment]
I agree! I would love to see more cursors! :)

**Comment by lmccart** [comment]
I think it might make sense to update the documentation and update native entering of strings, rather than adding more constants for each. this way we're not always falling behind. for example:
```js
cursor('s-resize');
cursor('arrow');
```

 it would be good to also include a link to the CSS cursor property reference page in the reference entry.

**Comment by lmccart** [comment]
this has been addressed with #3306
